debian/rules

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/make -f
+#
+# Copyright 2023 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+%:
+	dh $@
+
+override_dh_install:
+	./scripts/fetch-and-run-installer.sh "1.20.3" "/usr" "debian/tmp"
```

Comment:
[nit] go 1.20.4 was recently released, it has some security fixes.  Not sure if we care about staying on new go versions or not, but thought I'd mention it.  See https://go.dev/doc/devel/release